dts/x86/intel/elkhart_lake.dtsi

```diff
@@ -41,6 +41,15 @@
 		interrupt-controller;
 	};
 
+	intc_loapic: loapic@fee00000  {
+		compatible = "intel,loapic";
+		reg = <0xfee00000 0x1000>;
+		num-irqs = <6>;
```

Comment:
Is this `num-irqs` used?

Reply:
`num-irqs` is not used currently in the driver. The reason for adding here as it is characteristics of local APIC.  In my opinion, this value can be used in the future as described above.

Reply:
Then if we do not use it probably we can remove it

Reply:
Please look at the new patch. Removed `num-irqs`